packages/ab-testing/src/index.ts

```diff
@@ -25,29 +30,76 @@ function getModuloValue(experiment: string, userId: number | string): number {
     return crc32.calculate(String(userId), crc32.calculate(experiment)) % 100;
 }
 
+function validateAllocationCriteria(criteria: ForceInclude, userProfile: UserProfile): boolean {
+    let itemMatchesCriteria = true;
+    for (const key in criteria) {
+        if (!criteria[key].includes(userProfile[key])) {
```

Comment:
Is there a use case for doing the logical `AND` here?

It should stay consistent with `force_include` to avoid any confusion.

Reply:
NVM it makes sense under the context of the config file